pkg/test/framework/components/namespace/kube.go

```diff
@@ -101,6 +107,20 @@ func claimKube(ctx resource.Context, nsConfig *Config) (Instance, error) {
 	return &kubeNamespace{name: nsConfig.Prefix}, nil
 }
 
+// setNamespaceLabel labels a namespace with the given key, value pair
+func (n *kubeNamespace) setNamespaceLabel(key, value string) error {
+	// need to convert '/' to '~1' as per the JSON patch spec http://jsonpatch.com/#operations
+	jsonPatchEscapedKey := strings.ReplaceAll(key, "/", "~1")
```

Comment:
nit: if you used StrategicMergePatchType or one of the other ones I think you could avoid this